nationalarchives/ds-wagtail
# DF-708:Discovery links for record details

Ticket URL: https://national-archives.atlassian.net/browse/DF-708

## About these changes

- Added various discovery links for various records.

## How to check these changes

- Navigate to Search catalogue
- Select a record from - various buckets, or supply iaid in the url - http://127.0.0.1:8000/catalogue/id/{iaid}/
- Click "View this record page in Discovery, it should take you to the same record in Discovery.

## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
